exports/taskfiles/utils/misc.yaml

```diff
@@ -41,3 +43,47 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  reset-dir:
+    desc: "Clears the directory at the given path without removing the directory itself."
+    internal: true
+    label: "{{.TASK}}:{{.DIR_PATH}}"
+    requires:
+      vars: ["DIR_PATH"]
+    run: "when_changed"
```

Comment:
If `run:once` produces the same behaviour (because we set `label`), let's use `run:once` as it is easier to read/understand.